source/features/no-duplicate-list-update-time.tsx

```diff
@@ -9,7 +9,7 @@ function parseTime(element: HTMLElement): number {
 
 function init(): void {
 	for (const issue of select.all('.js-navigation-item[id^="issue_"]')) {
-		const [stateChangeTime, updateTime] = select.all('relative-time', issue);
+		const [stateChangeTime, updateTime] = select.all<'relative-time'>('relative-time', issue);
```

Comment:
???

Reply:
`@typescript-eslint/no-unsafe-argument`

Reply:
Read the rule description:

> This rule disallows calling a function with `any` in its arguments

Either the rule is faulty or our types are faulty. This piece of code has no meaning and should be undone. If it also breaks `querySelector` it might be worth reporting it on https://github.com/g-plane/typed-query-selector, who could know better what's happening